Problem1.java

```diff
@@ -1 +1,23 @@
-
+//Time Complexity = O(logn)
```

Comment:
define n

Reply:
I forgot to update it, will keep in mind next time. n is the size of array.